src/Symfony/Component/Workflow/Tests/StateMachineTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Symfony\Component\Workflow\Tests;
+
+use Symfony\Component\Workflow\Definition;
+use Symfony\Component\Workflow\Marking;
+use Symfony\Component\Workflow\StateMachine;
+use Symfony\Component\Workflow\Transition;
+
+class StateMachineTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @expectedException \Symfony\Component\Workflow\Exception\LogicException
+     */
```

Comment:
Can you add a `expectedExceptionMessage XXX`?
BTW, it should be a InvalidDefinitionException  instead of a LogicException ?